datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/glossary/mappers/GlossaryTermSnapshotMapper.java

```diff
@@ -17,27 +19,29 @@
  *
  * To be replaced by auto-generated mappers implementations
  */
-public class GlossaryTermSnapshotMapper implements ModelMapper<GlossaryTermSnapshot, GlossaryTerm> {
+public class GlossaryTermSnapshotMapper implements ModelMapper<EntityResponse, GlossaryTerm> {
```

Comment:
You can rename this to GlossaryTermMapper, now that there's no snapshot involved!

Reply:
sure will do

Reply:
done